packages/stepper/src/StepIcon.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { StepStates, StepIconProps, Mode } from './types';
+import CheckmarkIcon from '@leafygreen-ui/icon/dist/Checkmark';
+import EllipsisIcon from '@leafygreen-ui/icon/dist/Ellipsis';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+import { Overline } from '@leafygreen-ui/typography';
+import { useStepperContext } from './StepperContext';
+
+const Icon = ({ state, content }: StepIconProps) => {
+  if (state === StepStates.Completed) {
+    return <CheckmarkIcon />;
+  } else if (
+    state === StepStates.UpcomingMultiple ||
+    state === StepStates.CompletedMultiple
+  ) {
+    return <EllipsisIcon />;
+  } else {
+    // if Current (single) or Upcoming (single)
+    // TODO: Use centralized, reusable font-weight value
```

Comment:
Do you mean something like `tokens.fontWeight.medium` etc?

Reply:
Yeah. I know we've gone back and forth on this one a bunch though haha.

I think another issue here is the fact that our Typography components' APIs aren't flexible enough for this markup to be clean. In Mui, this would be something like `<Typography variant="body1" weight="medium" />` which feels nicer.